### ISSUE TITLE
Mounting frame for second PCB side

### ISSUE THREAD

**Issue by Kampi** [jakkra/ZSWatch-HW#30]
**Idea**

3D printed frame to provide some additional height during the PCB assembling. The PCB will placed upside down (with the components) into the frame and then you can add the stencil on the second side (or solder manually). After assembling you can put the PCB into an oven or use a hot air gun to solder the second side.

**Comment by blubbsy** [comment]
do i get the idea right? 

you want to have the negative of one soldered PCB-Side 3d printed to use it as a base to solder more comfortable the other side?

**Comment by Kampi** [comment]
Not exactly. A simple frame where you can put in the PC is enough

![image](https://github.com/user-attachments/assets/ced35aa9-9258-4cd0-aa1a-8829f73feaa7)

**Comment by blubbsy** [comment]
so basically it is a frame with PCB outline + little more for a border?
just thinking if this is something that can be automatized somehow with FreeCAD, but in general sounds like a ~5 minute CAD work.

**Comment by Kampi** [comment]
Yep, it's not much work

**Comment by blubbsy** [comment]
Like this? 
![grafik](https://github.com/user-attachments/assets/f855e8fe-c445-4729-bc4d-3a7d22116afc)
![grafik](https://github.com/user-attachments/assets/59f33a69-246e-4247-85be-653f590be2aa)

Only have to add maybe an additional offset to not have a tight fit of the PCB in it. Put 1mm offset below the PCB and 1mm for the outer ring. Dont think that much of more is necessary for what i understood you would like to have this. I would keep it that way and if we notice that it is not really stable on a table surface, we simply change the base to have 3 or 5 legs.

(Havent done it with FreeCAD, FreeCAD killed me trying to re-use the PCB geometry...)

[Halterung.zip](https://github.com/user-attachments/files/17356405/Halterung.zip)

**Comment by Kampi** [comment]
Yeah - but I would like to have some plateau to mount the SMD stencil. You put the stencil on the PCB, adjust it with tape on the frame, add the solder paste, flip the stencil upwards, and remove the PCB. So you have to make sure that the PCB isn´t mounted that tight and maybe add a hole on the side to remove the PCB with the paste and the components from the frame.

![image](https://github.com/user-attachments/assets/a2a3529e-0885-46fe-bef0-42b426a06d49)

**Comment by blubbsy** [comment]
This way or do you prefer to have some contact points outside the PCB-Outline? 

![grafik](https://github.com/user-attachments/assets/1ba70c0d-e731-4758-a0e4-aae44e5eef3c)

The "plateau" is 5mm in the screenshot to actually be able to move the stencil (the transparent piece) a little bit and added a cut-out (5*5mm) to make life easier to lift the PCB again out of the holder. 

Was thinking to actually use the big through holes as positioning help, but then i though that maybe a FDM 3d printer would have some difficulties to make them properly and would rather prefer that the printer simply does the outline nicely.

**Comment by WenchongWang** [comment]
这是来自QQ邮箱的假期自动回复邮件。邮件已经收到，我会尽快回复您。
